src/model/SwimmingPoolIndoor.cpp

```diff
@@ -377,6 +375,22 @@ namespace model {
       return result;
     }
 
+    ComponentType SwimmingPoolIndoor_Impl::componentType() const {
+      return ComponentType::None;
+    }
+
+    std::vector<FuelType> SwimmingPoolIndoor_Impl::coolingFuelTypes() const {
+      return {FuelType::Electricity};
```

Comment:
Why does a Swimming Pool have a cooling fuel type?

Reply:
agree, I would consider a pool a zone load, not a piece of intentional conditioning equipment.